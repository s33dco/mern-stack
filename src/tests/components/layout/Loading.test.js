import React from 'react';
import { shallow } from 'enzyme';
import Loading from '../../../components/layout/Loading';

let wrapper;

beforeEach(() => {
	wrapper = shallow(<Loading />);
});

describe('Home component', () => {
	it('should render Loading component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
