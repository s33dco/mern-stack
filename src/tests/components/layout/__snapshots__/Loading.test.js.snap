// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render Loading component correctly 1`] = `ShallowWrapper {}`;
